src/pages/steps/AdditionalInformationStep/ContactInfoStep.tsx

```diff
@@ -113,7 +115,7 @@ const ContactInfoStep = ({
     });
 
     setContactInfoState(contactInfoArray);
-  }, [contactInfo, handleChangeCompleted]);
+  }, [contactInfo, onChangeCompleted]);
```

Comment:
I think this change will create an infinite loop. Because onChangeCompleted comes from the props and will be seen as a new. function reference each time.

Reply:
@simon-debruijn  done ✅